solution02/src/test/resources/user.properties

```diff
@@ -59,8 +59,8 @@ Gpassword=google123
 Grole=roleGoogle
 Gpermission=/permission/admin
 Alias=oauth2/token
-GoogleClientId=708789899168-enn4ur2oeq6dcuso264fdvhmv2frbljj.apps.googleusercontent.com
-GoogleSecret=vYdreldU_BFUkLKfGTm68Dcl
+GoogleClientId=708789899168-57bei6m30t5jj92gah7k9i7gc0epardh.apps.googleusercontent.com
+GoogleSecret=ggZVeaG3OWS0de8BTabdtucG
```

Comment:
IMO we should not have the client id and secret here. This should be blank and needs to documented.